pkg/cloud/aws/services/ec2/instances.go

```diff
@@ -191,6 +213,11 @@ func (s *Service) createInstance(machine *actuators.MachineScope, bootstrapToken
 		}
 
 		input.UserData = aws.String(userData)
+
+	default:
+		errMsg := fmt.Sprintf("Unknown node role %q", machine.Role())
+		klog.Error(errMsg)
+		return nil, fmt.Errorf(errMsg)
```

Comment:
we don't need to klog this deep, just return an error and please use the `github.com/pkg/errors` library